pkgs/development/python-modules/robotframework-databaselibrary/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, robotframework
+}:
+
+buildPythonPackage rec {
+  version = "1.2.4";
+  pname = "robotframework-databaselibrary";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "627d872b3dda6a308a650ac9e676dadedf9c294e4ef70ad207cbb86b78eb8847";
+  };
+
+  # unit tests are impure
+  doCheck = false;
+
+  propagatedBuildInputs = [ robotframework ];
+
+  meta = with stdenv.lib; {
+    description = "Database Library contains utilities meant for Robot Framework";
```

Comment:
Do you mind adding yourself as a maintainer?

Reply:
I agree. I'm hesitant to add packages that no one wants to maintain. But if you're willing to be a maintainer for this, I approve.